symposion/templates/reviews/_review_table.html

```diff
@@ -132,4 +153,27 @@
         });
     });
 </script>
+
+<script type="text/javascript">
+$(document).ready(function() {
+    var table = $("table.table-reviews").DataTable();
```

Comment:
It ought to be possible to put all this new Javascript inside the existing Javascript function (just above this).